core/src/main/java/org/geysermc/geyser/session/GeyserSession.java

```diff
@@ -424,6 +424,8 @@ public class GeyserSession implements GeyserConnection, GeyserCommandSource {
     private boolean emulatePost1_16Logic = true;
     @Setter
     private boolean emulatePost1_18Logic = true;
+    @Setter
+    private boolean oldSmithingTable = false;
```

Comment:
I know there is the javadoc on OldSmithingTableTranslator, but there should be a javadoc here that explains it as well, or at least links to an explanation elsewhere

Reply:
e.g. "The server is pre-1.20 and viaversion will send a furnace as the old style smithing table - see OldSmithingTableTranslator"